site/src/main/java/uk/nhs/digital/ps/beans/Publication.java

```diff
@@ -21,4 +24,11 @@ public StructuredText getKeyFacts() {
 
         return new StructuredText(getProperty(PublicationBase.PropertyKeys.KEY_FACTS, ""));
     }
+
+    @HippoEssentialsGenerated(internalName = PublicationBase.PropertyKeys.BODY_SECTIONS)
+    public List<HippoBean> getBodySections() {
```

Comment:
There is a method `uk.nhs.digital.ps.beans.BaseDocument#getChildBeansIfPermitted` whose use would eliminate separate calls to `assertPropertyPermitted` and `getChildBeansByName`.